envi/config.py

```diff
@@ -180,10 +180,16 @@ def parseConfigOption(self, optstr):
             valstr = str(int(valstr, 16))
 
         if not (valstr.startswith('"') and valstr.endswith('"')):
-            try:
-                int(valstr)
-            except:
-                valstr = '"' + valstr + '"'
+            if valstr.lower() == 'true':
```

Comment:
i prefer to keep it as legit json as much as possible.  i could be mistaken, but this seems closer to legit json with this delta.  if we want a str 'true' we surround it with quotes.  if we want "json true" we don't.  this section wants to simply allow for a clarification of json primitives.  if i got it wrong, let's fix it... but that was my goal.
it occurs to me that i could have simply used "pass" instead of assigning to valstr for true/false and it should have been the same effect.  the point is, we don't want to wrap true/false in quotes if we don't actually want strings.